cocos/scripting/js-bindings/auto/jsb_cocos2dx_audioengine_auto.cpp

```diff
@@ -1,5 +1,5 @@
 #include "scripting/js-bindings/auto/jsb_cocos2dx_audioengine_auto.hpp"
-#if (CC_TARGET_PLATFORM == CC_PLATFORM_WINRT || CC_TARGET_PLATFORM == CC_PLATFORM_ANDROID || CC_TARGET_PLATFORM == CC_PLATFORM_IOS || CC_TARGET_PLATFORM == CC_PLATFORM_MAC || CC_TARGET_PLATFORM == CC_PLATFORM_WIN32)
+#if (CC_TARGET_PLATFORM == CC_PLATFORM_OHOS || CC_TARGET_PLATFORM == CC_PLATFORM_WINRT || CC_TARGET_PLATFORM == CC_PLATFORM_ANDROID || CC_TARGET_PLATFORM == CC_PLATFORM_IOS || CC_TARGET_PLATFORM == CC_PLATFORM_MAC || CC_TARGET_PLATFORM == CC_PLATFORM_WIN32)
```

Comment:
https://github.com/cocos2d/cocos2d-x/blob/v3/tools/tojs/cocos2dx_audioengine.ini#L10C21-L10C26

This file should also be updated.

Reply:
The same as other `_auto` suffix files.

Reply:
![image](https://github.com/cocos2d/cocos2d-x/assets/57482234/ebbcb9e6-33f2-46e3-a79b-5c1e684d1f11)

Are you saying we need modify all the files under tojs and tolua like this?

Reply:
We haven't used this tool, so we haven't focused on this part before.

Reply:
Yes. `*_auto.cpp` files are generated by `tolua/tojs` tools automatically.

Reply:
ok